packages/atomic/cypress/support/commands.ts

```diff
@@ -1,6 +1,6 @@
 // Must be declared global to be detected by typescript (allows import/export)
-
-import {AnalyticsTracker, SearchEventRequest} from '../utils/analyticsUtils';
+import {SearchEventRequest} from '@coveo/headless/node_modules/coveo.analytics/src/events';
+import {AnalyticsTracker} from '../utils/analyticsUtils';
```

Comment:
Not sure why this was working before. `SearchEventRequest` is not exported from `analyticsUtils`, and exporting it causes an error due to it being an import from `src`.